fe/fe-core/src/main/java/org/apache/doris/catalog/HiveMetaStoreClientHelper.java

```diff
@@ -212,10 +213,10 @@ public static String normalizeS3LikeSchema(String location) {
     }
 
     private static String getAllFileStatus(List<TBrokerFileStatus> fileStatuses,
-            List<RemoteIterator<LocatedFileStatus>> remoteIterators, BlobStorage storage) throws UserException {
-        boolean needFullPath = storage.getStorageType() == StorageBackend.StorageType.S3
-                || storage.getStorageType() == StorageBackend.StorageType.OFS
-                || storage.getStorageType() == StorageBackend.StorageType.JFS;
+            List<RemoteIterator<LocatedFileStatus>> remoteIterators, RemoteFileSystem fs) throws UserException {
+        boolean needFullPath = fs.getStorageType() == StorageBackend.StorageType.S3
```

Comment:
Avoid this kind of code. Because if we add a new type of FS later, we may miss this place.
You can add a new interface `fs.needFullPath()` and handle it in the interface.